### PR TITLE
refactored student-view

### PR DESCRIPTION
**Pull request by rkzedrick**

### CODE REVIEW

File: student-information-management-application/src/main/java/com/student/information/management/app/facade/student/impl/StudentFacadeImpl.java [hunk 1]
```diff
@@ -12,6 +12,12 @@ public class StudentFacadeImpl implements StudentFacade {
 
     @Override
     public List<Student> getAllStudents() { return studentDao.getAllStudents(); }
+
+    @Override
+    public List<Student> getAllStudentById() {
```
Comment (esdi-00) [inline student-information-management-application/src/main/java/com/student/information/management/app/facade/student/impl/StudentFacadeImpl.java:17]: why null?

File: student-information-management-application/src/main/java/com/student/information/management/data/student/dao/impl/StudentDaoImpl.java [hunk 2]
```diff
@@ -19,24 +20,35 @@ public class StudentDaoImpl implements StudentDao {
...
             }
             return students;
+
         } catch (Exception e) {
             throw new RuntimeException(e);
         }
     }
+
+    @Override
+    public List<Student> testGetStudentById() {
```
Comment (esdi-00) [inline student-information-management-application/src/main/java/com/student/information/management/data/student/dao/impl/StudentDaoImpl.java:38]: same. why null?

**Review by esdi-00 — CHANGES_REQUESTED** [review]
please see comments

File: student-information-management-application/src/main/java/com/student/information/management/data/student/dao/StudentDao.java [hunk 3]
```diff
@@ -7,6 +7,7 @@
 
 public interface StudentDao {
     List<Student> getAllStudents();
+    List<Student> testGetStudentById();
```
Comment (RoCS2024) [inline student-information-management-application/src/main/java/com/student/information/management/data/student/dao/StudentDao.java:10]: why is there a test method here?

File: student-information-management-application/src/main/java/com/student/information/management/app/facade/student/StudentFacade.java [hunk 4]
```diff
@@ -6,7 +6,10 @@
 
 public interface StudentFacade {
     List<Student> getAllStudents();
-    Student getStudentById(String student_id);
+
+    List<Student> getAllStudentById(String studentId);
```
Comment (RoCS2024) [inline student-information-management-application/src/main/java/com/student/information/management/app/facade/student/StudentFacade.java:10]: check method name

**Review by RoCS2024 — APPROVED** [review]

**Review by esdi-00 — APPROVED** [review]

**Pull request merged**